docs/wiki/viewing-elpis-training-log-file.md

```diff
@@ -1,48 +1,30 @@
 # Viewing Elpis training log file
 
-During the training stage a log file is written to /state/models/XXXX/train.log. Note that this file isn't created as soon as the Start Training button is clicked, there is a slight delay while some data prep is done. To view this file:
+During the training stage a log file is written to `/state/models/XXXX/train.log`. Note that this file isn't created as soon as the "Start Training" button is clicked, there is a slight delay while some data prep is done. To view this file:
```

Comment:
I'm not sure if this is right- but is it `state/of_origin` now?

Reply:
yes! Good catch, thanks